cpp/src/arrow/compute/row/encode_internal.cc

```diff
@@ -158,8 +158,7 @@ Status RowTableEncoder::EncodeSelected(RowTableImpl* rows, uint32_t num_selected
   EncoderOffsets::GetRowOffsetsSelected(rows, batch_varbinary_cols_, num_selected,
                                         selection);
 
-  RETURN_NOT_OK(rows->AppendEmpty(static_cast<uint32_t>(0),
-                                  static_cast<uint32_t>(rows->offsets()[num_selected])));
+  RETURN_NOT_OK(rows->AppendEmpty(static_cast<uint32_t>(0), static_cast<uint32_t>(0)));
```

Comment:
Can we add parameter names to make sure we understand what's being passed?
```suggestion
  RETURN_NOT_OK(rows->AppendEmpty(/*xxx=*/ static_cast<uint32_t>(0), /*yyy=*/ static_cast<uint32_t>(0)));
```

Reply:
Yes. Will do.

Reply:
Done.